iree/compiler/Dialect/Flow/Transforms/Passes.cpp

```diff
@@ -122,62 +122,65 @@ void buildFlowTransformPassPipeline(OpPassManager &passManager,
                                     const TransformOptions &transformOptions) {
   // Special case peephole optimizations.
   FunctionLikeNest(passManager)
-      .addPass(createConvertConv2D1x1ToMatmulPass)
+      .addPass(IREE::Flow::createConvertConv2D1x1ToMatmulPass)
       .addPredicatedPass(clEnableConvToImg2Col,
-                         createConvertConv2DToImg2ColPass)
+                         IREE::Flow::createConvertConv2DToImg2ColPass)
       // Input should now be legal.
-      .addPass(createVerifyInputLegalityPass)
+      .addPass(IREE::Flow::createVerifyInputLegalityPass)
       // Catch matmul ops before we do anything else with them.
       .addPredicatedPass(
           !clMmt4dTargetOptions.empty(),
           []() {
-            return createConvertLinalgMatmulToMmt4DPass(clMmt4dTargetOptions);
+            return IREE::Flow::createConvertLinalgMatmulToMmt4DPass(clMmt4dTargetOptions);
           })
       // Pad linalg ops
       .addPredicatedPass(clEnablePaddingLinalgOps, []() {
-        return createPadLinalgOpsToIntegerMultiplePass(clLinalgOpsPaddingSize);
+        return IREE::Flow::createPadLinalgOpsToIntegerMultiplePass(clLinalgOpsPaddingSize);
```

Comment:
clang-format wants these lines to wrap

https://github.com/google/iree/runs/5336633750?check_suite_focus=true